src/main/resources/static/js/popup/ConnectBase.js

```diff
@@ -0,0 +1,23 @@
+class ConnectBase {
+  constructor() {
+    this._url = "http://localhost:3000/auth/";
+    this.headers = {
+      "Content-Type": "application/json"
```

Comment:
Should we compile you code or it gonna work without it? 
If it needs to be compiled so it becomes additional server and you have to put additional headers for CORS requests.
If not - everything will work fine.